mon-pix/app/components/navbar-header.js

```diff
@@ -1,5 +1,12 @@
+import { inject as service } from '@ember/service';
 import Component from '@glimmer/component';
 
+const FRENCH_DOMAIN_EXTENSION = 'fr';
+
 export default class NavbarHeader extends Component {
+  @service currentDomain;
 
+  get isFrenchDomainExtension() {
```

Comment:
L'intention est de savoir si on doit afficher ou non la Marianne. 

Pourquoi ne pas retranscrire cette intention dans le nom de la méthode plutôt que de donner une transcription technique ?

```suggestion
  get shouldShowTheMarianne() {
```